model-optimizer/unit_tests/mo/utils/versions_checker_test.py

```diff
@@ -86,6 +87,22 @@ def test_append_version_list(self):
         for i, v in enumerate(req_list):
             self.assertEqual(v, ref_list[i])
 
+    def test_append_version_list_sys_neg(self):
+        v1 = "mxnet>=1.7.0; sys_platform != 'win32'"
+        req_list = list()
+        parse_and_filter_versions_list(v1, req_list, {})
+        ref_list = [('mxnet', '>=', '1.7.0')] if sys.platform != 'win32' else []
+        for i, v in enumerate(req_list):
+            self.assertEqual(v, ref_list[i])
+
+    def test_append_version_list_sys(self):
+        v1 = "mxnet>=1.7.0; sys_platform == 'linux'"
+        req_list = list()
+        parse_and_filter_versions_list(v1, req_list, {})
+        ref_list = [('mxnet', '>=', '1.7.0')] if sys.platform != 'win32' else []
```

Comment:
On Mac the `sys.platform = 'darwin'`. So your tests will be failing on Mac OS.

Use unittest mock functionality to return needed value from `sys.platform` and fix the tests

Reply:
The condition in test was broken, I have fixed it.

Reply:
If you still think that mock functionality is necessary here, please reopen this discussion.